src/notification/email.ts

```diff
@@ -6,12 +6,23 @@ import nodemailer from 'nodemailer';
 
 const email = Config.notifications.email;
 
-const transporter = nodemailer.createTransport({
-	auth: {
-		pass: email.password,
-		user: email.username
-	},
-	service: 'gmail'
+const transportOptions: any = {};
+
+if (email.username && (email.password || email.smtpAddress)) {
+	transportOptions.auth = {};
+	transportOptions.auth.user = email.username;
+	transportOptions.auth.pass = email.password;
+}
+
+if (email.smtpAddress) {
+	transportOptions.host = email.smtpAddress;
+	transportOptions.port = (email.smtpPort ? email.smtpPort : 25);
```

Comment:
```suggestion
	transportOptions.port = email.smtpPort;
```

This is fine because of the defaulting done in `config.ts`.

Reply:
As long as your set your environment variable ot `SMTP_PORT=""` or removing that line, it should be returning `25` since you're using `envOrNumber(process.env.SMTP_PORT, 25)`.

Reply:
right, yea that is exactly what I would expect would happen, however I am getting a returned value of `0` for some reason... I just checked on a couple other Config items that also use `envOrNumber` and have a default specified and I am getting `0` for those as well..

Reply:
Interesting... I did the same and didn't get those results which is strange. What OS and are you running in terminal or IDE?

Reply:
Wow, that is so weird. I am running Windows 10 on IDE (WebStorm)... Oh WAIT... I wonder if its because I am running Node 12.14 instead of 14...

Reply:
You'll want to update your branch with `main`, this was fixed in #364

Reply:
Yea I think that's what the problem is. I went ahead and commited the recommendation.